src/views/section.ts

```diff
@@ -0,0 +1,92 @@
+/*!
+ * © 2019 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  PAGE_BREAK_AFTER,
+  PAGE_BREAK_BEFORE,
+  PAGE_BREAK_BEFORE_AND_AFTER,
+} from '@manuscripts/transform'
+
+import { PluginState, sectionTitleKey } from '../plugins/section_title'
+import { BaseNodeProps } from './base_node_view'
+import BlockView from './block_view'
+import { createNodeView } from './creators'
+
+// handle sections numbering after track-changes process
+export const handleSectionNumbering = (sections: PluginState) => {
+  sections.forEach((sectionNumber, sectionId) => {
+    const section = document.getElementById(sectionId)
+    const sectionTitle = section?.querySelector('h1')
```

Comment:
This logic affecting the view of the section_title should be in the section_title and affecting its view directly.